### PR TITLE
[WIP] Docs Redesign

### PR DESCRIPTION
**Pull request by m-abdelwahab**
## What does this PR do?

This PR is a revamp for the entire docs website.  Homepage, docs page, and blog.  It's a WIP and there are still things to be added/changed

### CODE REVIEW

File: src/components/SDKs.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+export const SDKs = () => {
+  return (
+    <section className="my-20">
+      <h2 className="text-3xl md:text-4xl mb-2">SDKs</h2>
+      <p className="mb-8">
+        Find SDK tutorials and API references in the language of your choice.
+      </p>
+      <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6 lg:gap-8">
+        <Link to={useBaseUrl("/docs/concepts/introduction")}>
```
Comment (sw-yx) [inline src/components/SDKs.js:15]:
```suggestion
        <Link to={useBaseUrl("/docs/node/introduction")}>
```

**Review by sw-yx — APPROVED** [review]
looks like /application-development was deleted, be sure to add in a redirect

**Review by sw-yx — CHANGES_REQUESTED** [review]
netlify looks like it is failing on the glossary page also disappearing.

```
12:00:53 AM:   ✖ FAIL load build/docs/glossary
12:00:53 AM:   | operator: load
12:00:53 AM:   | expected: 200 build/docs/glossary
12:00:53 AM:   |   actual: ENOENT: no such file or directory, open '/opt/build/repo/build/docs/glossary'
12:00:53 AM:   |       at: build/404.html:20:4905 <a class="footer__link-item" href="/docs/glossary">...</a>
12:00:53 AM:   ✖ FAIL load build/blog/tags/bugs
12:00:53 AM:   | operator: load
12:00:53 AM:   | expected: 200 build/blog/tags/bugs
12:00:53 AM:   |   actual: ENOENT: no such file or directory, open '/opt/build/repo/build/blog/tags/bugs'
12:00:53 AM:   |       at: build/blog/index.html:20:21716 <a class="mr-2 my-2 no-underline inline-flex items-center px-3 py-0.5 rounded-full text-sm font-medium bg-[color:var(--ifm-badge-background-color)] text-[color:var(--ifm-color)] hover:opacity-80" href="/blog/tags/bugs">...</a>
```

File: src/components/Intro.js [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+          />
+          <path
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="M11.5223 8.36487L6.75835 3.08342C7.97767 2.68073 9.53838 3.15399 10.6381 4.37315C11.7378 5.5923 12.0482 7.19339 11.5223 8.36487ZM6.15533 8.43785L6.13426 8.4145C6.13776 8.4184 6.14126 8.4223 6.14478 8.42619C6.14829 8.43009 6.15181 8.43397 6.15533 8.43785Z"
+            fill="currentColor"
+          />
+        </svg>
+
+        <h3 className="text-xl font-semibold mb-4">System Tools</h3>
```
[the remark this answers is not in the document]
Comment (sw-yx) [reply src/components/Intro.js:179]: Yes. and "Dev Tools", not "Developer Tools" - its cleaner

**Review by flossypurse — APPROVED** [review]

**Pull request merged**